ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_percentile.py

```diff
@@ -0,0 +1,60 @@
+# global
```

Comment:
This should be in test_order_statistics.py not in its own file.

Reply:
Hello, thank you very much for the feedback, I will start working on this but I just have one question:
Should this 'percentile' task be attached to this issue #8953, like the median for example?

![image](https://user-images.githubusercontent.com/108529319/209542032-8d736740-31c6-4f0a-afb6-f8bb764ff034.png)